admin/services/check-start.service.js

```diff
@@ -29,14 +29,14 @@ const checkStartService = {
     if (found) {
       throw new Error(`Failed to generate a unique UUID for the check code.  Pupil [${pupilId}]`)
     }
-
     // Save the details to the `Check` collection
     const check = new Check({
       pupilId,
       checkCode,
       checkWindowId: checkWindow._id,
       checkFormId: checkForm._id,
-      pupilLoginDate: new Date()
+      pupilLoginDate: new Date(),
```

Comment:
elsewhere we are using moment.  I would like to see consistency across the entire app

pretty sure we have a date service?

Reply:
@updated to use moment. As stated earlier date service is used for formatting so it shouldn't be required.